test/__tests__/warehouse.test.js

```diff
@@ -1138,12 +1139,57 @@ describe("validTimestamp", () => {
     {
       input: '05:23:59.244Z',
       expected: false,
+    },
+    {
+      input: {abc: 123},
+      expected: false,
+    },
+    {
+      input: {
+        toString: '2023-06-14T05:23:59.244Z'
+      },
+      expected: false,
     }
   ]
 
-  for (const testCase of testCases) {
-    it(`should return ${testCase.expected} for ${testCase.input}`, () => {
+  for (let i = 0; i < testCases.length; i++) {
+    const testCase = testCases[i];
+    it(`should return ${testCase.expected} for testcase ${i + 1}`, () => {
       expect(validTimestamp(testCase.input)).toEqual(testCase.expected);
     });
   }
 });
+
+describe("isBlank", () => {
+  const testCases = [
+    {
+      input: null,
+      expected: true
+    },
+    {
+      input: "",
+      expected: true
+    },
+    {
+      input: "test",
+      expected: false
+    },
+    {
+      input: 1634762544,
+      expected: false
+    },
+    {
+      input: {
+        toString: '2023-06-14T05:23:59.244Z'
+      },
+      expected: false,
+    },
+  ]
+
+  for (let i = 0; i < testCases.length; i++) {
+    const testCase = testCases[i];
+    it(`should return ${testCase.expected} for testcase ${i + 1}`, () => {
```

Comment:
```suggestion
    it(`should return ${testCase.expected} for ${testCase.name}`, () => {
```
nit: assign a name to test instead of numericals